docs/recommendations/DNA/substitution.md

```diff
@@ -4,17 +4,24 @@
 
 Substitution: a sequence change where, compared to a reference sequence, **one** nucleotide is replaced by **one** other nucleotide.
 
-## Description
-
-Format: `<prefix><position_substituted><reference_nucleotide><new_nucleotide>`
-
-Example: `g.123A>G`
-
-- `prefix` = type of reference sequence used
-- `<position_substituted>` = position nucleotide sustituted = 123
-- `<reference_nucleotide>` = nucleotide at reference position = A
-- `>` = type of change is a substitution = >
-- `<new_nucleotide>` = substituted nucleotide = G
+## Syntax
+
+<table class="syntax">
+<tr>
+<th>Syntax</th>
+<td><code>sequence_identifier ":" coordinate_type "." position reference_nucleotide ">" alternate_nucleotide</code></td>
+</tr>
+<tr>
+<th>Examples</th>
+<td><code>NC_000001.11:g.123A>C</code></td>
+</tr>
+</table>
+
+- `sequence_identifier` = an identifier for DNA sequence
+- `position` = a single nucleotide position or inclusive range of deleted sequence
```

Comment:
This can not contain a range, but it has to be a single position. I struggled with the definitions and decided to go for `position` here while I used `positions` where variants can have a single position as well as a range. However, I used `positions` as well for insertions and inversions, that _must_ have a range. I like your separate definition for that, although `position_point` may suggest it's one position only. How about `interbase_position` for insertions and `position_range` for inversions? And, possibly, `single_position` for substitutions?